src/fetch-mock.js

```diff
@@ -161,6 +164,16 @@ FetchMock.addRoute = function (route) {
 		throw new Error('.mock() must be passed configuration for a route')
 	}
 
+  	const foundIndex = this.routes.findIndex(registeredRoute => {
```

Comment:
`findIndex` is in the ES6 standard so it should transpile ok.

Reply:
is this supported in node 4? I avoid using a lot of ES201* features in order to avoid the need to transpile for node

Reply:
looks like Travis runs with Node 4 so I think we're good.